Winget-AutoUpdate/Winget-Upgrade.ps1

```diff
@@ -164,22 +164,17 @@ if (Test-Network) {
             Write-Log "No new update." "Green"
         }
 
-        #Run WAU in user context if currently as system
-        if ($IsSystem) {
+        #Run WAU in user context if currently as system and the user task exist
+        $UserScheduledTask = Get-ScheduledTask -TaskName "Winget-AutoUpdate-UserContext" -ErrorAction SilentlyContinue
+        if ($IsSystem -and $UserScheduledTask) {
 
             #Get Winget system apps to excape them befor running user context
             Get-WingetSystemApps
 
             #Run user context scheduled task
-            $UserScheduledTask = Get-ScheduledTask -TaskName "Winget-AutoUpdate-UserContext" -ErrorAction SilentlyContinue
-            if ($UserScheduledTask) {
-                Write-Log "Starting WAU in User context"
-                Start-ScheduledTask $UserScheduledTask.TaskName -ErrorAction SilentlyContinue
-                Exit 0
-            }
-            else {
-                Write-Log "User context execution not installed"
```

Comment:
I would keep that log. Can be useful if someone doesn't understand why user context doesn't work.